running-pixelfed/installation.md

```diff
@@ -99,17 +99,15 @@ If you are running Redis on another machine:
 
 If you are using a Unix socket for Redis, then:
 
-- Set `REDIS_HOST` to the path of the socket, e.g. `/run/redis/redis.sock`
-- Set `REDIS_PORT` to `null`
-
-If you are using a Unix socket 
+- Set `REDIS_SCHEME` to `unix`
```

Comment:
REDIS_SCHEME and REDIS_PATH are only for using Predis instead of phpredis

Reply:
Hrm. I set `REDIS_HOST` to the Unix socket, like the docs said, and I got a stack trace about that being unexpected. Can't recall what exactly it said. I had to follow the proposed instructions here to get it to work. Bug?